assets/packages/adminbasics/build/adminbasics.js

```diff
@@ -29781,6 +29781,31 @@
         if (e.target != this) return;
         $('#accordion').find('.panel-collapse.collapse').not('#' + $(this).attr('id')).collapse('hide');
       });
+    },
+
+    /**
+     * Validates that an end date is not lower than a start date
+     */
+    validateEndDateHigherThanStart: function validateEndDateHigherThanStart(startDatePicker, endDatePicker, errorCallback) {
+      if (!startDatePicker || !startDatePicker.date()) {
```

Comment:
if startDatePicker didn't have date() function : it broke validation and form can not be submitted.
no ?